CHANGELOG.md

```diff
@@ -20,8 +20,8 @@ efficiency, reliability and portability.
   `p2wpkh` addresses.
 
```

Comment:
We should probably also mention the new `filterindex` here as well.

Reply:
ah, I'll move it here from the bottom, makes more sense its a big change.

Reply:
Okay, may not necessarily have need to have been removed from there. This works though.